### PR TITLE
added enum to boolean

### PR DESCRIPTION
**Pull request by amitu**
Use case is mentioned in #14.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ elm-stuff
 elm.js
 index.html
 node_modules
+yarn.lock
```
Comment (amitu) [inline .gitignore]: Shouldn't we be checking this file in? It will further reduce chances of improper versions of node packages.

Comment (jwoudenberg) [inline .gitignore]: Sounds good to me!

**Comment by amitu** [comment]
I was wondering whats up with that. At first I was like "thou must alwayst indenteth", but I maybe this is better. Good thing don't have have to think about such things in elm.

**Comment by jwoudenberg** [comment]
Awesome, thanks so much for this!

**Comment by jwoudenberg** [comment]
Published as 2.0.0.

**Comment by amitu** [comment]
Ah! I am working on min- and max-properties for object right now. 

PS: Why not rebase-merge?

**Comment by jwoudenberg** [comment]
I don't have a strong preference for the merge strategies. I think having a merge commit pointing back to a PR is kind of nice for the record though.

**Comment by amitu** [comment]
At work I prefer squash merge. Github inserts pull request number in commit title. All the intermediary commit messages are mostly noise, and connecting a commit to a pull request is also hard if we merge normally (merge commit mentions PR number, but individual commits don't). 

Github has done enough to make sure squash commit still shows the primary contributors photo etc in commit log, so attribution is preserved. 

Only downside is commit signing/verification, they get lost.

**Pull request merged**